controllers/provisioners/eks/helpers_test.go

```diff
@@ -31,6 +32,54 @@ import (
 	"github.com/pkg/errors"
 )
 
+func TestAutoscalerTags(t *testing.T) {
+	var (
+		k       = MockKubernetesClientSet()
+		ig      = MockInstanceGroup()
+		asgMock = NewAutoScalingMocker()
+		iamMock = NewIamMocker()
+		eksMock = NewEksMocker()
+		ec2Mock = NewEc2Mocker()
+	)
+
+	w := MockAwsWorker(asgMock, iamMock, eksMock, ec2Mock)
+
```

Comment:
You can probably use same getters suggested above in your tests, e.g. ig.GetAnnotations, configuration.GetLabels, etc